app/controllers/api/logins_controller.rb

```diff
@@ -1,20 +1,27 @@
 class Api::LoginsController < Api::ApplicationController
   skip_before_action :authenticate
+  before_action :create_debate_auth_token
 
   def create
-    debate = Debate.find_by code: params[:code]
-    return render json: { error: 'Debate not found' }, status: :not_found unless debate
     form = MobileUserForm.new(::MobileUser.new)
-    if form.validate(user_params.merge(auth_token: debate.auth_tokens.create!))
-      form.save(validate: false)
-      render json: { auth_token: form.auth_token.value }
+    if form.validate(user_params.merge(auth_token: auth_token))
+      form.save
+      render json: { auth_token: auth_token }
     else
-      render json: { error: form.errors.full_messages.join('. ') }, status: :bad_request
+      render json: { error: form.error_messages }, status: :bad_request
     end
   end
 
   private
 
+  attr_reader :auth_token
```

Comment:
Hmm, dziwnie mi wygląda stosowanie tutaj `attr_reader`. W kontrolerach jest to raczej mylące, proponuję pozostać przy zmiennych instancyjnych.

Chyba, że jest jakiś konkretny powód dla którego został użyty `attr_reader`, którego nie widzę?